azure-pipelines.yml

```diff
@@ -0,0 +1,150 @@
+trigger:
+- release/*
+  
+pool:
+  vmImage: windows-latest
+
+parameters:
+- name: doPublish
+  default: false
+  type: boolean
+- name: toDeploy
+  default: false
+  type: boolean
+
+variables:
+- name: buildConfiguration
+  value: 'Release'
+
+resources:
+  repositories:
+  - repository: inss-devops-common-lib
+    type: github
+    endpoint: InsolvencyService
+    name: InsolvencyService/inss-devops-common-lib
+    ref: develop
+    
+stages:
+- stage: BuildTest
+  displayName: 'Build and Test'
+  jobs:  
+    - job: 
+      steps:
+      - task: UseDotNet@2
+        inputs:
+          version: '6.0.x'
+
+      - task: DotNetCoreCLI@2
+        displayName: NuGet Restore
+        inputs:
+          command: 'restore'
+          projects: '**/INSS.FIP.sln'
+  
+      - task: DotNetCoreCLI@2
+        displayName: 'Build Solution'
+        inputs:
+          command: 'build'
+          arguments: '--configuration $(buildConfiguration)'        
+
+      - task: DotNetCoreCLI@2
+        displayName: 'Run Tests'
+        inputs:
+          command: test
+          projects: '**/*Tests/*.csproj'
+          arguments: '--configuration $(buildConfiguration) --collect "Code coverage"'
+
+      - task: DotNetCoreCLI@2
+        displayName: 'Publish Project'
+        inputs:
+          command: 'publish'
+          publishWebProjects: false
+          projects: '**/*.csproj'
+          arguments: '--configuration $(BuildConfiguration) --output $(Build.ArtifactStagingDirectory)'
+        condition: eq('${{ parameters.doPublish}}', 'true')
+
+      - task: PublishPipelineArtifact@1
+        displayName: 'Publish Artifact'
+        inputs:
+          targetPath: '$(Build.ArtifactStagingDirectory)' 
+          artifactName: 'INSS.FIP'
+        condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+
+- stage: DeployDev
+  displayName: 'Deploy to Dev'
+  condition: and(and(succeeded(), ne(variables['Build.Reason'], 'PullRequest')), eq('${{ parameters.toDeploy}}', 'true'))
+  variables:
+    - group: FIP-Dev
+  jobs:
+    - template: deploy-jobs-template.yml
+      parameters:
+        environment: 'Dev'
+        phaseBanner: $(PhaseBanner)
+        environmentBannerText: $(EnvironmentBannerText)
+        fipApiConnectorClientOptions__BaseAddress: $(FipApiConnectorClientOptions__BaseAddress)
+        iirwebdbContextConnectionString: $(IirwebdbContextConnectionString)
+        openApi__Version: $(OpenApi__Version)
+        openApi__DocVersion: $(OpenApi__DocVersion)
+        openApi__DocTitle: $(OpenApi__DocTitle)
+        appZipFile: 'INSS.FIP.zip'
```

Comment:
This is always the same, so doesn't seem much point in supplying it every time

Reply:
Removed in latest commit